Modules/Core/Common/include/itkRGBAPixel.h

```diff
@@ -81,7 +81,9 @@ class ITK_TEMPLATE_EXPORT RGBAPixel : public FixedArray<TComponent, 4>
 #ifdef ITK_FUTURE_LEGACY_REMOVE
   RGBAPixel() = default;
 #else
-  RGBAPixel() { this->Fill(0); }
+  constexpr RGBAPixel()
+    : Superclass()
```

Comment:
With this [force-push](https://github.com/InsightSoftwareConsortium/ITK/compare/7544cbd5a0302f9d53c60bf536e595288c7bafc4..6464b887f18b156562c1c21d6146b1af7f1beb2a), replacing `{}` with `()`, I'm hoping to address the warnings at https://open.cdash.org/viewBuildError.php?type=1&buildid=10096694 saying:

> In file included from [Modules/Core/Common/test/itkNumericTraitsTest.cxx:42](https://github.com/InsightSoftwareConsortium/ITK/blob/7544cbd5a0302f9d53c60bf536e595288c7bafc4/Modules/Core/Common/test/itkNumericTraitsTest.cxx#L42):
/.../s/Modules/Core/Common/include/itkNumericTraitsRGBPixel.h: In function 'void numeric_traits_test::CheckVariableLengthArrayTraits(const T&) [with T = itk::RGBPixel<char>]':

[Modules/Core/Common/include/itkNumericTraitsRGBPixel.h:118](https://github.com/InsightSoftwareConsortium/ITK/blob/7544cbd5a0302f9d53c60bf536e595288c7bafc4/Modules/Core/Common/include/itkNumericTraitsRGBPixel.h#L118):17: warning: '<anonymous>' may be used uninitialized in this function [-Wmaybe-uninitialized]

Reply:
It appears that with the old [Linux GCC 9.4.0](https://open.cdash.org/builds/10097099), the `constexpr` default-constructors that I wrote just don't properly zero-initialize the components, as they should. I'll think of another workaround...

Reply:
Another attempt, initializing by the seemingly redundant expression `Superclass(Superclass())`: [force-push](https://github.com/InsightSoftwareConsortium/ITK/compare/6464b887f18b156562c1c21d6146b1af7f1beb2a..07ee85ae52a882bdc634cf93fd3304e026d6bac0)

Reply:
Now might be a reasonable time to reconsider which compiler versions we support in ITK 6?

Reply:
@dzenanz It looks like this particular bug (as I reproduced with the example code at https://github.com/InsightSoftwareConsortium/ITK/pull/5087#discussion_r1889314541) is fixed with GCC 12.1 (which was released in May 2022): https://godbolt.org/z/of6Tdbzcj  GCC 11 still appears to have the compiler bug. I'm not sure if we can already drop GCC 11 now. Ubuntu-22.04 still comes with GCC 11.4 🤷 

Having said so, I think we have a workaround for this one now 😃